### DISCUSSION TITLE
Calendar direction

### DISCUSSION THREAD

**Discussion by angelicablirio** [onesine/react-tailwindcss-datepicker#85] [Q&A]
Hello, I would like to know how do I adjust the direction that opens the calendar. It's opening upwards, but I would like it to open downwards from the input.

Another question is how do I leave the months side by side, they are getting one below the other, as shown in the photo.

![image](https://user-images.githubusercontent.com/85036081/224023269-8cd98101-a140-4ea1-b375-27405e35c734.png)

Thank you very much for your attention!

**Comment by onesine** [comment]
Hi @angelicablirio 👋,
Sorry for the delay.

Thanks for using this software and thanks for your comments.

Normally, the calendars are displayed one under the other from a screen width of 767. This behavior should be ideal, because on small screens there is not enough space. We can always allow you to control this behavior. 

The opening of the calendar at the bottom also depends on the position of the date picker in the dom. The more space there is at the top, the calendar will be displayed at the top. The more space at the bottom, the calendar is displayed at the bottom. We can also let you control this behavior.
